pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -49,7 +49,7 @@ For cloud installations, scaling out can be limited by resource quotas.
 
 - [`kubectl`][kubectl] on your installation machine
 
-- For multi-cluster environments (Enterprise license): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
+- For customers running on a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
```

Comment:
@alembiewski is this a correct way to describe it? Or do you think we need to mention Managed Clusters specifically?

Reply:
Looks good to me!